pkg/preparer/orchestrate_test.go

```diff
@@ -218,9 +244,47 @@ func testPreparer(t *testing.T, f *FakeStore) (*Preparer, *fakeHooks, string) {
 	hooks := &fakeHooks{}
 	p.hooks = hooks
 	p.store = f
+	p.podStatusStore = &FakeStatusStore{}
 	return p, hooks, podRoot
 }
 
+func TestInstallWhiteListPodsInTheIntent(t *testing.T) {
+	whiteListPods := make(map[types.PodID]bool)
+	whiteListPods["hello"] = true
+
+	store := &FakeStore{
+		currentManifest: testManifest(t),
+		wipedReality:    true,
+	}
+
+	p, _, fakePodRoot := testPreparer(t, store)
+	defer p.Close()
+	defer os.RemoveAll(fakePodRoot)
+
+	err := p.InstallWhiteListPods(whiteListPods)
+	Assert(t).IsTrue(err != nil, "Pod cannot be installed.")
```

Comment:
Is this test asserting that an error occurs? If so, can the name of the test be changed to reflect that it is testing an unhappy path?